packages/containers/src/DeleteResource/DeleteResource.container.js

```diff
@@ -36,15 +36,10 @@ export default class DeleteResource extends React.Component {
 	 * Return the label and a boolean to confirm that the item has been found.
 	 * @param {object} resourceInfo
 	 */
-	getLabel(resourceType, id) {
-		let resource;
-		const resourcesCollection = this.context.store.getState().cmf.collections.get(resourceType);
-		if (resourcesCollection) {
-			resource = resourcesCollection.find(
-				resoureCollection => resoureCollection.get('id') === id,
-			);
-		}
-		return resource ? { label: resource.get('label'), found: true } : { label: '', found: false };
+	getLabel() {
+		return this.props.resource
+			? { label: this.props.resource.get('label'), found: true }
+			: { label: '', found: false };
 	}
```

Comment:
return {
label: this.props.resource.get('label', '') 
found: !!resource
}
what do u think ?